src/index.html

```diff
@@ -8,6 +8,52 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="all-stars stars--0">
+      <span class="stars__star"></span>
```

Comment:
rename your block to just 'stars' (according to BEM block is stars and it's element is stars_star)

Reply:
have not  learned the BEM topic yet